apps/src/templates/sectionProgress/SummaryViewLegend.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { PropTypes, Component } from 'react';
+import ProgressBox from './ProgressBox';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+
+const styles = {
+  legendBox: {
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    borderColor: color.lightest_gray,
+    backgroundColor: color.white,
+    float: 'left',
+    display: 'inline-block',
+    marginTop: 20,
+  },
+  progressBox: {
+    float: 'left',
+    padding: 10,
+    width: 130,
+    marginLeft: 10,
+  },
+  heading: {
+    fontSize: 18,
+    fontWeight: 'bold',
+    color: color.charcoal,
+    float: 'left',
+    paddingTop: 10,
+    paddingBottom: 10,
+    textAlign: 'center',
+  },
+  label: {
+    fontSize: 14,
+    fontWeight: 900,
```

Comment:
I've never seen us use font-weight with gotham, only the `5r` syntax like this: https://github.com/code-dot-org/code-dot-org/blob/80fe1ac6cdda80567f620d3e7bcffda54a1bd079/apps/src/tutorialExplorer/tutorial.jsx#L42

Not sure if this matters. Definitely something we can clean up in later PR.

Reply:
I used 'bold' because I was following the example in ProgressLegend, https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/progress/ProgressLegend.jsx#L45

Reply:
ah looks like I'm wrong. It used to be that "bold" didn't do anything to gotham so we had to use the "5r" system, but it seems to work now 👍